filecoin-hashers/Cargo.toml

```diff
@@ -10,18 +10,20 @@ readme = "README.md"
 
 [dependencies]
 bellperson = "0.18.0"
-blstrs = "0.4.0"
+blstrs = { version = "0.4.0", features = ["gpu"] }
 generic-array = "0.14.4"
 merkletree = "0.21.0"
+ec-gpu = "0.1.0"
 ff = "0.11.0"
 anyhow = "1.0.34"
 serde = "1.0.117"
 rand = "0.8.0"
-
-neptune = { version = "5.1.0", optional = true, features = ["arity2", "arity4", "arity8", "arity11", "arity16", "arity24", "arity36"] }
+neptune = { git = "https://github.com/filecoin-project/neptune", branch = "fr-as-trait", optional = true, features = ["bls", "pasta", "arity2", "arity4", "arity8", "arity11", "arity16", "arity24", "arity36"] }
```

Comment:
TODO: change `neptune` dependencies once [this PR](https://github.com/filecoin-project/neptune/pull/135) is merged.

Reply:
We decided to keep that branch for now until there's a proper `pasta_curves` release.